src/test/java/com/github/iveina2021/page/LoginPage.java

```diff
@@ -20,14 +33,32 @@ public class LoginPage {
     @FindBy(css = "button[data-qa='signup-button']")
     public SelenideElement signupButton;
 
+    @FindBy(xpath = "//p[contains(@style, 'color:')]")
```

Comment:
(bad selector) here you check that paragraph should contain color, it can be any paragraph with color on the screen

Reply:
Edited this selector